openy_block/modules/openy_block_date/src/StackMiddleware/BlockDateCacheInvalidator.php

```diff
@@ -17,6 +18,11 @@ class BlockDateCacheInvalidator implements HttpKernelInterface {
    */
   const TAG = 'block_date';
 
+  /**
```

Comment:
@podarok if we don't need to support backport and version 2.0.x of the module will be supported only by Drupal 10, we can remove this constant definition there.

Reply:
we don't need a BC  here @mkdok

Reply:
but it's ok